examples/first-frame/main.cpp

```diff
@@ -41,9 +42,35 @@
 #endif
 #include <ios>
 #include <iostream>
+#include <map>
 
 using namespace aditof;
 
+static const char Help_Menu[] =
+    R"(First-frame usage:
+    first-frame [-config | --CONFIG <CONFIG>] 
+    first-frame (-h | --help)
+    first-frame [-ip | --ip <ip>] [-m | --m <mode>] [--config | --CONFIG <CONFIG>]
+
+    Arguments:
+      [-config | --CONFIG <CONFIG>]            Input config_default.json file (which has *.ccb and *.cfg)
```

Comment:
`-config` is also mismatch from the previous line, `--config`

Reply:
Typo's for mandatory argument <CONFIG>in the help menu will be solved once the parser update will be merged (mandatory arguments will no longer need a prefix of type "-"/"--" so they will be deleted)